src/__tests__/__snapshots__/Footer.js.snap

```diff
@@ -0,0 +1,18 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`renders Footer correctly 1`] = `
+<footer
+  className="footer"
+>
+  <div
+    className="container"
+  >
```

Comment:
- [ ] when I tried to run your application on my local machine I got the following error while installing dependencies please make sure to solve that error I see that the first reviewer had the same experience
- I think the problem is we have a different version of the node that's was caused the error it happened to me when I was a student 
- the error I got 
![Screen Shot 2022-10-08 at 10 29 55](https://user-images.githubusercontent.com/61790587/194701045-4e462472-0adf-41b2-851b-4e16874ad1b5.png)

Reply:
use yarn .

yarn install and then yarn start.
it works on my side , no error .

Reply:
@rebel216  leave a comment to next reviewer in order to user yarn

Reply:
it works on my side. :(